Loop/Settings/SettingsView.swift

```diff
@@ -47,8 +48,18 @@ struct SettingsView: View {
                 .frame(width: 500)
                 .frame(minHeight: 500, maxHeight: 680)
 
-            MoreSettingsView()
+            ExcludeListSettingsView()
                 .tag(5)
+                .tabItem {
+                    Image(systemName: "xmark.rectangle")
+                    Text("Black list")
+                }
+                .environmentObject(appListManager)
+                .frame(width: 500)
+                .frame(minHeight: 500, maxHeight: 680)
```

Comment:
Unlike the keybindings tab, this tab only contains a list, and no toggle buttons etc.
So I think this might look better without a minimum height (but keep the max height)!